### PR TITLE
Assets for 1.28

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by yabirgb** [comment]
Complete diff with all the changes in the db. It includes:

- new assets
- maker ilk registry update
- yearn vaults updates
- curve registry updates

[diff.txt](https://github.com/rotki/rotki/files/11482026/diff.txt)

File: rotkehlchen/assets/exchanges_mappings/kraken.py [hunk 1]
```diff
@@ -223,4 +223,8 @@
     evm_address_to_identifier('0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'TBTC',  # noqa: E501
     evm_address_to_identifier('0x5283D291DBCF85356A21bA090E6db59121208b44', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'BLUR',  # noqa: E501
     'HDX': 'HDX',
+    evm_address_to_identifier('0xC581b735A1688071A1746c968e0798D642EDE491', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'EURT',  # noqa: E501
+    evm_address_to_identifier('0xfc5A1A6EB076a2C7aD06eD22C90d7E710E35ad0a', ChainID.ARBITRUM, EvmTokenKind.ERC20): 'GMX',  # noqa: E501
+    'SUI': 'SUI',
+    # evm_address_to_identifier('0xcf0C122c6b73ff809C693DB761e7BaeBe62b6a2E', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'FLOKI',  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/assets/exchanges_mappings/kraken.py:229]: Why is this commented out?

```suggestion
    # evm_address_to_identifier('0xcf0C122c6b73ff809C693DB761e7BaeBe62b6a2E', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'FLOKI',  # noqa: E501
```

File: rotkehlchen/tests/api/test_database.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def test_query_db_info(rotkehlchen_api_server, data_dir, username, start_with_lo
     response = requests.get(api_url_for(rotkehlchen_api_server, 'databaseinforesource'))
     result = assert_proper_response_with_result(response)
     assert len(result) == 2
-    assert result['globaldb'] == {'globaldb_assets_version': 16, 'globaldb_schema_version': 5}
+    assert result['globaldb'] == {'globaldb_assets_version': 18, 'globaldb_schema_version': 5}
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_database.py:34]: why did it jump by 2?

File: rotkehlchen/tests/exchanges/test_gemini.py [hunk 3]
```diff
@@ -22,6 +22,8 @@
 from rotkehlchen.types import AssetMovementCategory, Location, Timestamp
 from rotkehlchen.utils.misc import ts_now
 
+UNSUPPORTED_GEMENI_PAIRS = {'btcgusdperp'}
```
Comment (LefterisJP) [inline rotkehlchen/tests/exchanges/test_gemini.py:25]: dude!

```suggestion
UNSUPPORTED_GEMINI_PAIRS = {'btcgusdperp'}
```

Also why is this not supported?

File: rotkehlchen/assets/exchanges_mappings/poloniex.py [hunk 4]
```diff
@@ -140,8 +140,15 @@
     evm_address_to_identifier('0xb3999F658C0391d94A37f7FF328F3feC942BcADC', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'HFT',  # noqa: E501
     evm_address_to_identifier('0xbB95cc1c662D89822bda29D2e147B124406e6e42', ChainID.BINANCE, EvmTokenKind.ERC20): 'TRR',  # noqa: E501
     evm_address_to_identifier('0xaaAEBE6Fe48E54f431b0C390CfaF0b017d09D42d', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'CEL',  # noqa: E501
-    evm_address_to_identifier('0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa', ChainID.BINANCE, EvmTokenKind.ERC20): 'ALPACA',  # noqa: E501
+    evm_address_to_identifier('0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa', ChainID.BINANCE, EvmTokenKind.ERC20): 'TBTC',  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/assets/exchanges_mappings/poloniex.py:143]: Why did this change to TBTC? This seems wrong. That's not a contract in BSC

https://bscscan.com/address/0x8dAEBADE922dF735c38C80C7eBD708Af50815fAa

File: rotkehlchen/assets/exchanges_mappings/bitfinex.py [hunk 5]
```diff
@@ -70,5 +70,9 @@
     evm_address_to_identifier('0x2a3bFF78B79A009976EeA096a51A948a3dC00e34', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'WILD',  # noqa: E501
     evm_address_to_identifier('0x607F4C5BB672230e8672085532f7e901544a7375', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'RLC',  # noqa: E501
     evm_address_to_identifier('0x7659CE147D0e714454073a5dd7003544234b6Aa0', ChainID.ETHEREUM, EvmTokenKind.ERC20): 'XCAD',  # noqa: E501
+    evm_address_to_identifier('0xfb5B838b6cfEEdC2873aB27866079AC55363D37E', ChainID.BINANCE, EvmTokenKind.ERC20): 'FLOKI',  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/assets/exchanges_mappings/bitfinex.py:73]: you have 2 FLOKIs here.
```suggestion
```

File: rotkehlchen/assets/exchanges_mappings/poloniex.py (hunk 4 above)
Comment (yabirgb) [reply rotkehlchen/assets/exchanges_mappings/poloniex.py:143]: Saw that TBTC was missing and when adding it the address was already there but with the wronmg symbol. ALpaca is a few lines later

Comment (LefterisJP) [reply rotkehlchen/assets/exchanges_mappings/poloniex.py:143]: Please check the link of the contract. That's not contract. Address is wrong.

File: rotkehlchen/assets/exchanges_mappings/kraken.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/assets/exchanges_mappings/kraken.py]: is not needed. Removed it

**Review by LefterisJP — APPROVED** [review]
lgtm

**Pull request merged**